packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+
+API endpoints for creating, getting, and updating Policy and Rule objects function the same way in Okta Identity Engine as they do in the existing Okta `/policies` API. The API endpoints support the following operations:
+
+ * Get Policy
+ * List Policies
+ * Get Rule
+ * List Rules
+ * Create Policy
+ * Create Rule
+ * Update Policy
+ * Update Rule
+ * Delete Policy
+ * Delete Rule
+ 
+### Sample Operation: Get Policy
+
+<ApiOperation method="get" url="/policies/${PolicyID}" />
+
+Given a `PolicyID` identifier, returns a [Policy Object](#okta-identity-engine-policy-objects).
+
+#### Request Body
+
+| Property | Type   | Description                          |
+|----------|--------|--------------------------------------|
+| PolicyID | String | The unique identifier of the Policy. |
+
+#### Request Query Parameters
+
+None
+
+#### Response Body
+
+A [Policy Object](#okta-identity-engine-policy-objects).
+
+#### Usage Example
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/policies/${policyId}"
+```
+
+##### Response
+
+```json
+{
+    "id": "rs10xzYOyK9ux6v70g4",
+    "name": "Default Policy",
+    "type": "Okta:IdpRouting",
+    "status": "ACTIVE",
+    "default": true,
+    "_links": {
+        "self": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10xzYOyK9ux6v70g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "rules": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10xzYOyK9ux6v70g4/rules",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Syntax for All Other Operations
+
+See <https://developer.okta.com/docs/reference/api/policy/#policy-api-operations> for request syntax for all other operations.
+
+## Okta Identity Engine Policy Objects
+
+Okta Identity Engine introduces a set of new Policy and Rule objects that are different from the existing Policy and API objects:
+
+* [IdP Routing Policy Object](#idp-routing-policy-object)
+* [IdP Routing Rule Object](#idp-routing-rule-object)
+* [Identifier Match Policy Object](#identifier-match-policy-object)
+* [Identifier Match Rule Object](#identifier-match-rule-object)
+* [Unknown User Policy Object](#uknown-user-policy-object)
+* [Unknown User Rule Object](#unknown-user-rule-object)
+* [Sign On Policy Object](#sign-on-policy-object)
+* [Sign On Rule Object](#sign-on-rule-object)
+* [User Profile Policy Object](#user-profile-policy-object)
+* [User Profile Rule Object](#user-profile-rule-object)
+
+Default instances of each Policy type are created automatically when Okta Identity Engine is enabled for your org. Each of the default Policy objects also has a default rule that is created automatically.
```

Comment:
`default Policy objects also has a default rule` -> `default Policy objects also has a default Rule` consistency when talking about Policy vs Rules. Either both start with upper-case or none of them.